defi/src/utils/getProtocolTvl.ts

```diff
@@ -12,14 +12,18 @@ import {
 import getTVLOfRecordClosestToTimestamp from "./shared/getRecordClosestToTimestamp";
 
 const _getLastHourlyRecord = (protocol: Protocol) => getLastRecord(hourlyTvl(protocol.id))
+const _getLastTokensHourlyRecord = (protocol: Protocol) => getLastRecord(hourlyUsdTokensTvl(protocol.id))
 const _getYesterdayTvl = (protocol: Protocol) => getTVLOfRecordClosestToTimestamp(hourlyTvl(protocol.id), Math.round(Date.now() / 1000) - secondsInDay, secondsInDay)
 const _getLastWeekTvl = (protocol: Protocol) => getTVLOfRecordClosestToTimestamp(hourlyTvl(protocol.id), Math.round(Date.now() / 1000) - secondsInWeek, secondsInDay)
 const _getLastMonthTvl = (protocol: Protocol) => getTVLOfRecordClosestToTimestamp(hourlyTvl(protocol.id), Math.round(Date.now() / 1000) - secondsInWeek * 4, secondsInDay)
 
+const includeSection = (chainDisplayName:string)=> !extraSections.includes(chainDisplayName) && !chainDisplayName.includes("-")
+
 export async function getProtocolTvl(
   protocol: Readonly<Protocol>,
   useNewChainNames: boolean, {
     getLastHourlyRecord = _getLastHourlyRecord,
+    getLastTokensHourlyRecord = _getLastTokensHourlyRecord,
```

Comment:
also, need to pass this parameter as input from the function calling is, else instead of relying on pg/cache data, we will make ddb call each time